### ISSUE TITLE
Implement C-CDA Care Team Section

### ISSUE THREAD

**Issue by sjpadgett** [openemr/openemr#5513]
### Describe the problem
HL7 Rel2.1 Cures moved care team members from under the documentionOf template into it's own sectional template using an organizer to hold team members and stats.

Although the current testing passes as we are currently implementing and the care team section is a MAY include, seems to me this is important to be correctly implemented and used since our CDA is becoming pretty nice.
Bottom line, it just bugs me to not be current and correct. Well worth the $1.99 I'm charging!
### Describe the solution you'd like
Add the section. This can be tricky sometimes when new templates are involved.
